verticapy/utilities.py

```diff
@@ -67,42 +66,103 @@
 
 #
 # ---#
-def create_verticapy_schema(cursor=None):
+def create_table(
+    table_name: str,
+    dtype: dict,
+    schema: str = "",
+    temporary_table: bool = False,
+    temporary_local_table: bool = True,
+    genSQL: bool = False,
+):
     """
 ---------------------------------------------------------------------------
-Creates a schema named 'verticapy' used to store VerticaPy extended models.
+Creates a new table using the input columns names and data types.
```

Comment:
"the input columns" -> "the input column's" (missing apostrophe)

Reply:
"the input columns'" as it is still plural. It will be corrected in the next commit.